Cargo.toml

```diff
@@ -1,3 +1,6 @@
+[profile.release]
+debug = true
+
```

Comment:
Perhaps we could use a [custom profile](https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles):

```suggestion
# Release profile with debug symbols, useful for profiling
[profile.release-debug]
inherits = "release"
debug = true
```

That way debug symbols are only generated if desired.